pkg/drivers/kvm/kvm.go

```diff
@@ -348,18 +367,16 @@ func (d *Driver) Remove() error {
 	}
 	defer conn.Close()
 
-	//Tear down network and disk if they exist
-	log.Debug("Checking if the network needs to be deleted")
-	network, err := conn.LookupNetworkByName(d.PrivateNetwork)
+	// Tear down network if it exists and is not in use by another minikube instance
+	log.Debug("Trying to delete the networks (if possible)")
+	err = d.deleteNetwork()
```

Comment:
nit: inline the if statement here.

Reply:
fair enough, inlining this statement right now